### DISCUSSION TITLE
GitHub packager.

### DISCUSSION THREAD

**Discussion by dae-vinity** [TurboWarp/packager#682] [Ideas]
A GitHub bot that can get the .sb3 from one branch and package it into a website to another branch for GitHub pages.

If this is rejected, is there a way I could make this myself? (I know more than just Scratch I promise.)

**Comment by GarboMuffin** [comment]
We have a Node.js API you could use: https://github.com/TurboWarp/packager/tree/master/node-api-docs

For more examples, see: https://github.com/TurboWarp/packager/network/dependents

**Reply by dae-vinity** [reply]
Cool, thanks man.
